lib/image_db.rb

```diff
@@ -95,17 +105,17 @@ def full_update
     # http://www.flickr.com/services/api/misc.urls.html
     # NOTE: Page count verified 2015-07-22
     (1..30).each do |page|
-      p ({ :flickr => '42027916@N00', :page => page })
-      favorites = flickr.favorites.getPublicList(:user_id => '42027916@N00', :extras => 'url_n', :page => page).map {|p| "http://www.flickr.com/photos/#{p["owner"]}/#{p["id"]}"}
-      favorites.each {|l| @images.add l }
+      p ({ flickr: "42027916@N00", page: page })
```

Comment:
`(...)` interpreted as grouped expression.<br>Do not write to stdout. Use Rails' logger if you want to log.